sqlglot/parser.py

```diff
@@ -5390,6 +5393,9 @@ def _parse_alias(
         )
 
         if alias:
+            if any_token is None and alias.this.upper() in self.EXCLUDED_IMPLICIT_ALIAS_IDS:
```

Comment:
if not any_token

Reply:
why don't we just remove FORMAT from id_var why do we need excluded_implicit_alias_ids?

Reply:
From my testing, `FORMAT` is a valid identifier in all cases _except_ when it serves as an 'implicit' alias, so in a way Clickhouse allows this middle state. Removing it from `ID_VAR` will solve this issue but it will disallow it's general identifier use